introspection/meson.build

```diff
@@ -53,23 +53,31 @@ ifaces = [
 
 ifaces_xmls = []
 sources = []
+introspection_files = []
 
 # FIXME?: DBUS_INTERFACE_DOCS/docbook files are generated in
 # "meson.current_build_dir()" instead of "meson.source_root()/docs/api"
 foreach iface: ifaces
   iface_xml = iface + '.xml'
   ifaces_xmls += files(iface_xml)
 
-  sources += gnome.gdbus_codegen(
+  res = gnome.gdbus_codegen(
     iface,
     iface_xml,
     interface_prefix: 'org.freedesktop.NetworkManager',
     namespace: 'NMDBus',
     docbook: 'dbus'
   )
 
-  # FIXME: gnome.gdbus_codegen should return the docbook as a target result
+  sources += res
+
   content_files += join_paths(meson.current_build_dir(), 'dbus-' + iface_xml)
+
+  # FIXME: res is an array only since 0.46. Documentation won't work
+  # with older versions
```

Comment:
`FIXME` makes it sound like there would be something to fix. But there isn't. The issue is handled as best as it could. Either `s/FIXME:/Note:/` or `s/FIXME: //`. Alternatively, the fixme comment should vaguely describe how the fix looks like.